server/services/config/conditions/v1/conditions.ts

```diff
@@ -14,8 +14,18 @@ enum GroupName {
   RESTRICTED = 'Restricted additional conditions',
 }
 
-export const getPersistedAbuseAndBehaviours = (conditions: AdditionalConditions) =>
-  (conditions as AdditionalConditionsV1)?.COMPLYREQUIREMENTS?.abuseAndBehaviours
+export const modifyAdditionalConditions = (someConditions: AdditionalConditions) => {
```

Comment:
before this used to just pass back the list of behaviours so that the licence builder could prefix a space to each of the items (barring the first)

As we now have the same requirement for a list of professions but only for v2, I've moved the modification logic into each version of the conditions so it can vary per version